android/.gitignore

```diff
@@ -0,0 +1,13 @@
+gradle-wrapper.jar
+/.gradle
+/captures/
+/gradlew
+/gradlew.bat
+/local.properties
+GeneratedPluginRegistrant.java
+
+# Remember to never publicly share your keystore.
+# See https://flutter.dev/docs/deployment/android#reference-the-keystore-from-the-app
+key.properties
+**/*.keystore
+**/*.jks
```

Comment:
This is part of the Flutter Android project template now. I would vote that we keep it, because it doesn't hurt and if we update the project again, it will receive potential updates made by flutter and we don't have to manually merge it into our own gitignore.

Reply:
It was always a part of the template, I just unified them, because I prefer that

Reply:
Ok, I will merge them.